solr/core/src/test-files/solr/configsets/query-limits/conf/solrconfig.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" ?>
+
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<config>
+  <metrics/>
+
+  <luceneMatchVersion>${tests.luceneMatchVersion:LATEST}</luceneMatchVersion>
+
+  <directoryFactory name="DirectoryFactory" class="${solr.directoryFactory:solr.RAMDirectoryFactory}">
+    <!-- used to keep RAM reqs down for HdfsDirectoryFactory -->
+    <bool name="solr.hdfs.blockcache.enabled">${solr.hdfs.blockcache.enabled:true}</bool>
+    <int name="solr.hdfs.blockcache.blocksperbank">${solr.hdfs.blockcache.blocksperbank:1024}</int>
+    <str name="solr.hdfs.home">${solr.hdfs.home:}</str>
+    <str name="solr.hdfs.confdir">${solr.hdfs.confdir:}</str>
+    <str name="solr.hdfs.blockcache.global">${solr.hdfs.blockcache.global:true}</str>
+  </directoryFactory>
+  <schemaFactory class="ClassicIndexSchemaFactory"/>
+
+  <indexReaderFactory name="IndexReaderFactory"
+       class="org.apache.solr.cloud.TrollingIndexReaderFactory"></indexReaderFactory >
+
+  <dataDir>${solr.data.dir:}</dataDir>
+
+  <!-- an update processor the explicitly excludes distrib to test
+       clean errors when people attempt atomic updates w/o it
+  -->
+  <updateRequestProcessorChain name="nodistrib" >
+    <processor class="solr.NoOpDistributingUpdateProcessorFactory" />
+    <processor class="solr.RunUpdateProcessorFactory" />
+  </updateRequestProcessorChain>
+
+  <requestHandler name="/select" class="solr.SearchHandler">
+  </requestHandler>
+
+  <updateHandler class="solr.DirectUpdateHandler2">
+    <updateLog>
+      <str name="dir">${solr.ulog.dir:}</str>
+    </updateLog>
+  </updateHandler>
+
+  <query>
+         <filterCache
+                 size="0"
+                 initialSize="0"
+                 autowarmCount="0"/>
+         <queryResultCache
+                  size="0"
+                  initialSize="0"
+                  autowarmCount="0"/>
+         <documentCache
+                   size="0"
+                   initialSize="0"
+                   autowarmCount="0"/>
+         <fieldValueCache
+                 size="0"
+                 autowarmCount="0"/>
+  </query>
+
+  <searchComponent name="expensiveSearchComponent"
+                   class="org.apache.solr.search.ExpensiveSearchComponent"/>
```

Comment:
Should this class `org.apache.solr.search.ExpensiveSearchComponent`  be under java/org/apache/solr/search directory? I see it under test directory. 
<img width="527" alt="image" src="https://github.com/apache/solr/assets/3736645/1da42f6b-8529-4c89-b4da-216e718e7888">